### DISCUSSION TITLE
DataTable not showing

### DISCUSSION THREAD

**Discussion by bucari** [rappasoft/laravel-livewire-tables#716] [Q&A]
Hi, i have installed laravel-livewire-tables and configured it to use bootstrap-5 but i can't make it work.

even with a simple example like this:
UsersTable.php

`<?php

namespace App\Http\Livewire;

use App\Models\User;
use Illuminate\Database\Eloquent\Builder;
use Rappasoft\LaravelLivewireTables\DataTableComponent;
use Rappasoft\LaravelLivewireTables\Views\Column;

class UsersTable extends DataTableComponent
{
    public function configure(): void {
        $this->setPrimaryKey('id');
    }

    public function columns(): array
    {
        return [
            Column::make('Name')
            ->sortable(),
            Column::make('email')
            ->sortable(),
        ];
    }

    public function query(): Builder
    {
        return User::query();
    }
}`

the view where i put the livewire component <livewire:users-table /> is broken and only a white page is shown

what am I doing wrong?

**Comment by edikurniawan-dev** [comment]
that's code works fine on me

**Reply by bucari** [reply]
ok

i'm using jetstream with jetstrap and laravel 9, there may be problems with them?

**Reply by edikurniawan-dev** [reply]
what the rappasoft/datatable version are you using?

**Reply by bucari** [reply]
i'm using version 2

**Reply by edikurniawan-dev** [reply]
in the UsersTable.php

`<?php

namespace App\Http\Livewire;

use Rappasoft\LaravelLivewireTables\DataTableComponent;
use Rappasoft\LaravelLivewireTables\Views\Column;
use App\Models\User;

class UsersTable extends DataTableComponent
{
    protected $model = User::class;

    public function configure(): void
    {
        $this->setPrimaryKey('id');
    }

    public function columns(): array
    {
        return [
            Column::make("Id", "id")
                ->sortable(),
            Column::make("Name", "name")
                ->sortable(),
            Column::make("Email", "email")
                ->sortable(),
            Column::make("Created at", "created_at")
                ->sortable(),
            Column::make("Updated at", "updated_at")
                ->sortable(),
        ];
    }
}`

**Reply by edikurniawan-dev** [reply]
I have tried using jetstrap and bootstrap5 and its work fine
